cuqi/distribution/_modifiedhalfnormal.py

```diff
@@ -7,14 +7,17 @@
 class ModifiedHalfNormal(Distribution):
     """
     Represents a modified half-normal (MHN) distribution, a three-parameter family of distributions generalizing the Gamma distribution.
-    The distribution is continuous with pdf 
-    f(x; alpha, beta, gamma) propto x^(alpha-1) * exp(-beta * x^2 + gamma * x)
+    The distribution is continuous with pdf
+
+    .. math::
+    
+        f(x; \\alpha, \\beta, \\gamma) \propto x^{(\\alpha-1)} * \exp(-\\beta * x^2 + \\gamma * x)

```

Comment:
one note here, using alpha for example as a word, shows the direct link with the parameters. However, having it as a symbol is much nicer. May I suggest in the definition of parameters below that you add the symbol to make the link explicit, for example:
```
    Parameters
    ----------
    alpha : float
        The polynomial exponent parameter :math:`\\alpha` of the MHN distribution. Must be positive.
    beta : float
        The quadratic exponential parameter :math:`\\beta` of the MHN distribution. Must be positive.
    gamma : float
        The linear exponential parameter :math:`\\gamma` of the MHN distribution.
```

Reply:
Done as suggested